### PR TITLE
Add Session C: Meshing with cjdns and messing with secure-scuttlebutt

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

**Comment by benhylau** [comment]
@dcwalk ready for merge when you approve

File: sessions/session-c.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Session C: Meshing with cjdns and messing with secure-scuttlebutt
+
+## Description
+
+This hands-on session begins with [configuring a Raspberry Pi](https://github.com/tomeshnet/prototype-cjdns-pi2) as a [cjdns](https://github.com/cjdelisle/cjdns) mesh node. Then we will enable the IP tunnel feature to route all Internet traffic to another cjdns node through the [Hyperboria](https://github.com/hyperboria) encrypted mesh network. If you have a Raspberry Pi 3, please bring that along with [Raspbian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/) pre-flashed, but that is not a prerequisite to participate and there will be limited number of Raspberry Pi's provided at the workshop.
```
Comment (dcwalk) [inline sessions/session-c.md]: Can we pull out the materials comment out either to the top or below this para and the scuttlebutt one?

File: sessions/session-c.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Session C: Meshing with cjdns and messing with secure-scuttlebutt
```
Comment (dcwalk) [inline sessions/session-c.md]: Dig this! But I think we need a short title (the second could be like the after-colon part?)
e.g. "Build a Mesh Node: Meshing with cjdns and messing with secure-scuttlebutt" and I'll use the short one on the website schedule?

File: sessions/session-c.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Description
+
+This hands-on session begins with [configuring a Raspberry Pi](https://github.com/tomeshnet/prototype-cjdns-pi2) as a [cjdns](https://github.com/cjdelisle/cjdns) mesh node. Then we will enable the IP tunnel feature to route all Internet traffic to another cjdns node through the [Hyperboria](https://github.com/hyperboria) encrypted mesh network. If you have a Raspberry Pi 3, please bring that along with [Raspbian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/) pre-flashed, but that is not a prerequisite to participate and there will be limited number of Raspberry Pi's provided at the workshop.
+
+We will run [secure-scuttlebutt](https://github.com/ssbc/secure-scuttlebutt) on our ad-hoc mesh, which keeps an append-only database on each node with peer-to-peer replication across the network, backing mesh-friendly applications such as [Patchwork](https://github.com/ssbc/patchwork) and [dnssb](https://github.com/ansuz/dnssb). Join us at this workshop for some DIY peer-to-peer social networking and DNS distribution!
```
Comment (dcwalk) [inline sessions/session-c.md]: unclear what 'backing' means here? Also could we link out to wiki or explain what DNS is (or expand the acronym and link out)?

File: sessions/session-c.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Session Objective
+
+- Use the cjdns protocol to set up an encrypted mesh network of Raspberry Pi's
```
Comment (dcwalk) [inline sessions/session-c.md:13]: Maybe link to cjdns here or on first use?

File: sessions/session-c.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- Configure an IP tunnel to exit all Internet traffic at a distant cjdns node, essentially having the Raspberry Pi serve as a VPN router
```
Comment (dcwalk) [inline sessions/session-c.md]: again, link for IP tunnel explanation and VPN?

File: sessions/session-c.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+## Length
+
+1:20
+
+## Materials
+
+- Laptop (not provided)
+- Raspberry Pi 3 kit (limited number provided)
+- TP-LINK TL-WN722N WiFi USB adapter
+- Ethernet cable
```
Comment (dcwalk) [inline sessions/session-c.md]: Are these all requests for participants? 
Can you list an estimate of kits you'll have? usb adapters, ethernet cable, etc...? 
then I can add in what we can borrow from semaphore!

**Review by dcwalk — CHANGES_REQUESTED** [review]
Thanks @benhylau! just a couple quick bits (sorry should have been clearer about stuff on the template)

**Comment by dcwalk** [comment]
Also whoa, just used the review bit and that was odd "changes requested" sounds too harsh :(

File: sessions/session-c.md (hunk 4 above)
Comment (benhylau) [reply sessions/session-c.md:13]: It was linked on first use.

**Review by dcwalk — APPROVED** [review]

File: sessions/session-c.md (hunk 4 above)
Comment (dcwalk) [reply sessions/session-c.md:13]: aha whoops! thx

**Pull request merged**